arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
@athipp - Thanks for the explanation. One thing to clarify is that RPs can put any number of transient/non-terminal states inside "provisioningState" property. Yes, there is a fixed set of terminal states as you mentioned above. W.r.t the new proposed property, I wanted to confirm one thing-

It looks like the resource will go into these new states outside of resource provisioning. Can you confirm? In this case, we are fine with the new property. However, please consider renaming it to something else like a "status" so that its not confused with provisioningState.

But if it was part of resource provisioning, there is a case for merging it with "provisioningState" and we can discuss this more in person.